src/components/global/navbars/RoleBasedNavbar/JudgesNavbar.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+
+import Link from 'next/link'
+import Image from 'next/image'
+import {usePathname} from 'next/navigation'
+import {LogOut} from 'lucide-react'
+
+import {Button} from '@/components/ui/Button'
+import bountyLabLogo from '@public/images/bounty-lab-icon.png'
+import {cn} from '@/lib/utils/client/tailwind'
+import {UserAvatar} from '@/components/ui/Avatar'
+import {PATHS} from '@/lib/utils/common/paths'
+import SignOutButton from '@/components/ui/SignOutButon'
+import RoleSwitch from '@/components/ui/RoleSwitch'
+
+const JudgesNavbar = () => {
+  const pathname = usePathname()
+
+  return (
+    <div className="pointer-events-none fixed z-10 flex w-full flex-col bg-black px-24 py-6">
+      <div className="pointer-events-auto flex flex-row items-center justify-between">
+        <nav className="flex items-center gap-10">
+          <Link href={PATHS.home}>
+            <Image
+              width={42}
+              height={42}
+              alt="Bounty Lab logo"
+              src={bountyLabLogo}
+            />
+          </Link>
+          <Button
+            asChild
+            variant="link"
+            className={cn(pathname === PATHS.judgeContests && 'font-bold')}>
+            <Link href={PATHS.judgeContests}>Contests</Link>
+          </Button>
+        </nav>
+        <div className="flex items-center gap-10">
+          <UserAvatar />
+          <RoleSwitch />
```

Comment:
If a judge switches his role, will he be able to select it back? We might want to split currently selected role in the DB from the actual JUDGE permissions 🤷

Reply:
Currently the judge role cannot be switched. This is not an issue currently since the judge role will be assigned internally and there is no use case where a judge user would want to switch roles. This will probably need to change in the future if we allow other non-internal users to be judges, but for now this is fine. 

Also the judge role has to be manually assigned by modifying the DB directly. I'm planning on adding an admin role/permission level that will allow an admin to modify this and assign the judge role to users. However, I would prioritize other more critical tasks before improving this.

Reply:
Alright, I was just asking because I've seen the `<RoleSwitch>` in the code, but I see now that it's showing nothing 👍